packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -318,7 +330,29 @@ export class Connector implements IConnector {
         }
 
         return new Promise((resolve, reject) => {
-            this._waitingForLoadingFinished.set(requestId, { reject, resolve });
+            /*
+             * There some cases where the a request never dispatch
```

Comment:
> Sometimes the `debugging protocol` doesn't dispatch the event `loadFinished` for a request even thought it's finished. To avoid waiting forever, timeout at `waitForContentLoaded` and continue the process.